client/views/forgotPassword/forgotPassword.js

```diff
@@ -0,0 +1,54 @@
+// Function for email validation
+function validateEmail(email) {
+  var reg = /^([A-Za-z0-9_\-\.])+\@([A-Za-z0-9_\-\.])+\.([A-Za-z]{2,4})$/;
+  if(reg.test(email) === false) {
+    console.log('Invalid email');
+    return false;
+  }
+  return true;
+}
+
+function showError(reason, parentElement, nextElement) {
+  if (typeof currentError !== 'undefined') {
```

Comment:
let's be consistent and put a space after `if` everywhere

Reply:
@peter-hank Add spaces everywhere after if conditional